src/Features/Core/Portable/SimplifyInterpolation/AbstractSimplifyInterpolationDiagnosticAnalyzer.cs

```diff
@@ -63,18 +64,16 @@ private void AnalyzeInterpolation(OperationAnalysisContext context)
                 return;
             }
 
-            var locations = ImmutableArray.Create(interpolation.Syntax.GetLocation());
+            context.ReportDiagnostic(DiagnosticHelper.Create(
+                UnnecessaryWithSuggestionDescriptor,
+                unnecessaryLocations.First(),
+                option.Notification.Severity,
+                additionalLocations: ImmutableArray.Create(interpolation.Syntax.GetLocation()),
+                properties: null));
 
-            var severity = option.Notification.Severity;
-
-            for (var i = 0; i < unnecessaryLocations.Length; i++)
+            for (var i = 1; i < unnecessaryLocations.Length; i++)
```

Comment:
```suggestion
            // We start at 1 because the 0th element was used above to make the main diagnostic descriptor.
            // All the rest are used to just fade out the correct portions of the user's code.
            for (var i = 1; i < unnecessaryLocations.Length; i++)
```